Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -5,6 +5,7 @@
 using Microsoft.MixedReality.Toolkit.Utilities;
 using System;
 using System.Collections.Generic;
+using System.Runtime.CompilerServices;
```

Comment:
What requires this?

Reply:
AT line 2206, it's basically to be like c++ inline keyword
[MethodImpl(MethodImplOptions.AggressiveInlining)]